package.json

```diff
@@ -35,7 +35,7 @@
     "lint": "eslint .",
     "precommit": "lint-staged && npm-run-all -p flow types",
     "prepare": "husky install && npm-run-all clean -p lib types",
-    "test": "nyc mocha 'test/**/*.test.js'",
+    "test": "nyc mocha 'test/**/*.test.ts'",
```

Comment:
Does mocha support TS without configuration now, or is it configured with sucrase or tsc already?

Reply:
The `.nycrc` file has the `sucrase/register` settings, so `nyc mocha` handles TypeScript without trouble.